cassandra/4.0/modules/ROOT/pages/cassandra-connector-examples.adoc

```diff
@@ -300,19 +300,1079 @@ payload]]></ee:set-payload>
 
 This example shows the following ways to manipulate data in an Apache Cassandra database:
 
-* Get table names from a keyspace.
+* <<get-table-names, Get table names from a keyspace.>>
 * Perform the following CRUD operations on data in an Apache Cassandra table:
-** Insert
-** Select
-** Update
-** Delete
+** <<insert, Insert>>
+** <<select, Select>>
+** <<update, Update>>
+** <<delete, Delete>>
 * Perform the following operations on an Apache Cassandra table column:
-** Delete the column's value.
-** Rename a column.
-** Add a new column.
-** Drop a column.
-* Execute a Cassandra Query Language (CQL) query.
-* Drop data from the Apache Cassandra database table.
+** <<delete-columns, Delete the column's value.>>
+** <<rename-columns, Rename a column.>>
+** <<add-new-column, Add a new column.>>
+** <<drop-columns, Drop a column.>>
+* <<execute-cql, Execute a Cassandra Query Language (CQL) query.>>
+* Drop data from the Apache Cassandra database table:
+** <<drop-table, Drop a table from a keyspace.>>
+** <<drop-keyspace, Drop a keyspace.>>
+
+[[get-table-names]]
+=== Get Table Names from a Keyspace
+
+This Mule flow returns all table names from the specified keyspace.
+
+image::get-table-names.png[Studio flow for the Get table names from keyspace operation]
+
+In this flow, you configure:
+
+* An *HTTP Listener* component
+* A *Get table names from keyspace* operation
+* A *Transform Message* component
+
+==== Configure HTTP Listener
```

Comment:
Consider for a future sprint: Since this section is repeated over and over with no changes, this would be a good candidate for reuse (a partial)! We could probably reuse this same section in many of our examples.

Reply:
This is good!! However we have to change the last bullet for each flow to a different path:
. Set the *Path* field to `/addNewColumn`. <---- this part 

Would we have a partial for everything up to that part then, and then manually add that last step after each partial?

Reply:
Yep!